README.md

```diff
@@ -201,6 +201,13 @@ var railsImagePath = "<%= helpers.image_path('rails.png') %>";
 
 This is enabled by the `rails-erb-loader` loader rule in `config/webpack/shared.js`.
 
+## Ready for Elm
```

Comment:
[nitpick] I think we maintain a order in this README (most recent integrations are added to the bottom), so I think this info should be added below `Ready for Vue` section